gammapy/image/maps.py

```diff
@@ -341,12 +341,13 @@ def reproject(self, refheader, mode='interp', *args, **kwargs):
         skymap : `SkyMap`
             Skymap reprojected onto ``refheader``.
         """
+
         from reproject import reproject_interp
         out = reproject_interp(
             input_data=self.to_image_hdu(),
             output_projection=refheader,
             *args,
-            **kwargs,
+            **kwargs
```

Comment:
@JouvinLea This is fixed in `master`. If you fetch the latest `master` version and rebase this pull request, you should get a merge conflict for this line. If you resolve it (picking the solution from master), the error should be gone.

Let me know if this works.